src/code-components/RichTextArea/RichTextArea.register.ts

```diff
@@ -212,6 +234,18 @@ export function registerRichTextArea(
         valueProp: "value",
         onChangeProp: "onChange",
       },
+      error: {
+        type: "readonly",
+        variableType: "object",
+        onChangeProp: "onError",
+        initVal: undefined,
+      },
+      loading: {
```

Comment:
Editor is loading? Or image upload is loading? Or text drop is loading? Rename the prop to avoid confusions.


```suggestion
     loadingImageUpload: {
```